doc: Add Bitbucket bad application id troubleshooting [CY-985] (#382)
codacy/chart: docs/configuration/integrations/bitbucket-server.md

    ```

After this is done you will be able to use Bitbucket Server to authenticate to Codacy.

## Troubleshooting

- During an authentication procedure, if you got stuck on the provider and this message is shown
  - ![Invalid client id](./bitbucket-invalid-client-id.png)
    It means you have not introduced the client id when configuring Bitbucket.com on Codacy.

>     - Make sure the value matches the one in your Bitbucket.com application

Reviewer: ```suggestion
    - Make sure the value matches the one in your Bitbucket Server application
```

Author: BB Server was not supposed to be committed. All the errors will be handled in our side.

    - If you still could not find the problem:

>         - Extract the parameter `client_id`, from the browser address bar in Bitbucket.com where the error appears (e.g.: `r8QJDkkxj8unYfg4Bd`)

Reviewer: ```suggestion
        - Extract the parameter `client_id`, from the browser address bar in your Bitbucket Server URL where the error appears (e.g.: `r8QJDkkxj8unYfg4Bd`)
```

Author: BB Server was not supposed to be committed. All the errors will be handled in our side.

>         - Check if the application id on Bitbucket.com matches this value

Reviewer: ```suggestion
        - Check if the application id on Bitbucket Server matches this value
```

Author: BB Server was not supposed to be committed. All the errors will be handled in our side.
